binding/Binding/Definitions.cs

```diff
@@ -1348,6 +1356,15 @@ public SKRect (float left, float top, float right, float bottom)
 			this.bottom = bottom;
 		}
 
+		public SKRect (SKPoint location, SKSize sizeI)
```

Comment:
The second parameter is of type `SKSize` but it is named sizeI, maybe change it to size so it's more intuitive.